### DISCUSSION TITLE
The overall design screenshots

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/gbowne1site_react#16] [General]
I also posted these in:

https://github.com/gbowne1/gbowne1site_react/pull/15

![Screenshot 2023-04-18 at 00-32-31 Portfolio Greg Bowne](https://user-images.githubusercontent.com/47549872/232705805-46f45d66-5143-40b2-9a66-ac5e8564b5ed.png)

![Screenshot 2023-04-18 at 00-25-38 Gregory Bowne](https://user-images.githubusercontent.com/47549872/232705868-c4ae5f11-63bb-4729-8002-63b5846d0085.png)

![Screenshot 2023-04-18 at 00-23-34 Gregory Bowne](https://user-images.githubusercontent.com/47549872/232705883-c35fbd83-c5d2-4665-bedf-6b765cc838fe.png)

![Screenshot 2023-04-18 at 00-18-09 Bootstrap Example](https://user-images.githubusercontent.com/47549872/232705889-4d3e2950-f2b2-4ee2-946a-4240ca0ea5ed.png)

**Comment by k-deepak04** [comment]
@gbowne1 got this design 👍 
will try my best to deliver this design :)
can we connect on other platform such as whatsapp or mail or instagram?. i would like to talk on different subject matter.

**Reply by gbowne1** [reply]
I would prefer to keep communications on here only @k-deepak04

**Reply by k-deepak04** [reply]
No problem @gbowne1 
i'll write my query here only.

**Comment by k-deepak04** [comment]
I'm a 3rd year Engineering student started doing open source now. As after few months i'll start applying for jobs so i want to build my resume or profile for it.
Can you help in that @gbowne1 
Like if i can get a testimonial from you or some sort of certificate saying that this person has contributed in my projects etc.
Any kind of help would be appriciated . Also i would like to know if there are any freelancing work at your firm or buisness so that i can add value to it.

**Reply by gbowne1** [reply]
I do not do resume or CV work. Sorry.

Anyone can see that you have contributed to my project(s). I can probably do a testimonial if you put up some sort of app or site, or use GitHub.

I do not have any freelancing work available for anyone. Sorry.

**Reply by k-deepak04** [reply]
Okay thanks @gbowne1
